manager/vulnerabilities_handler.py

```diff
@@ -51,10 +51,18 @@ def __init__(self, rh_account_id, list_args, query_args, uri, args):
             "status_id": SQL('status_id'),
             "status": SQL('status_id'),
         }
+        default_sort_columns = {
```

Comment:
Could you also add `public_date` filtering for `impact`? I think CVEs should be also sorted when filtering by impact.

Reply:
Make sense, thanks